src/components/Aside/Nav.jsx

```diff
@@ -1,13 +1,45 @@
 import React, { Component } from 'react';
-import { Link } from 'react-router-dom';
+import { NavLink } from 'react-router-dom';
 import styles from './Nav.module.scss';
 
+const activeStyle = {
+  backgroundColor: 'red',
+  borderRadius: '10px',
+  padding: '5px 10px',
+};
+
 export class Nav extends Component {
   render() {
     return (
       <nav className={styles.nav}>
-        <Link to="/">Home</Link>
-        <Link to="/counter">Counter</Link>
+        <NavLink
+          style={({ isActive }) => (isActive ? activeStyle : undefined)}
```

Comment:
move this fn to utils.
getActiveStyle = (isActive) => isActive ? activeStyle : undefined